app/client/components/delivery/address/deliveryAddressForm.tsx

```diff
@@ -175,12 +178,16 @@ const FormContainer = (props: FormContainerProps) => {
   const addressChangeInformation = Object.values(
     props.contactIdToArrayOfProductDetail
   )
-    .flatMap(babelFlatMapFunction)
-    .map(productDetail => {
-      const friendlyProductName = capitalize(
-        ProductTypes.contentSubscriptions.mapGroupedToSpecific?.(productDetail)
-          .friendlyName
-      );
+    .flatMap<ProductDetail>(flattenEquivalent)
+    .map(productDetail => ({
+      productDetail,
+      productType: ProductTypes.contentSubscriptions.mapGroupedToSpecific?.(
+        productDetail
+      )
+    }))
+    .filter(_ => _.productType && _.productType.delivery?.showAddress)
```

Comment:
filtering to products which actually themselves support delivery address update - effectively excluding 'Digital subscription' (a.k.a. digipack)